pycloudlib/azure/util.py

```diff
@@ -14,6 +15,19 @@
 )
 
 
+class AzureParams(NamedTuple):
+    """Azure Parameters Class."""
+
+    name: str
+    parameters: Optional[Dict[str, Any]]
```

Comment:
Add a comment explaining the purpose of this class and its attributes - specifically how it relates to the azure sdk.